posts/2019-04-11-Rust-1.34.0.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: post
+title: "Announcing Rust 1.34.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.34.0. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.34.0 is as easy as:
+
+```console
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the appropriate
+page on our website, and check out the [detailed release notes for 1.34.0][notes] on GitHub.
```

Comment:
The way this is written, it suggests that (only) people who don't have rust(up) already should check out the release notes. 

The opposite is probably true -- new people need not check out the release notes, people already familiar with Rust will benefit more from doing so.

Since the release notes are mentioned later, I'd suggest removing that part of the sentence here.